us.materialio.Materialious.yml

```diff
@@ -0,0 +1,55 @@
+app-id: us.materialio.Materialious
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
```

Comment:
I think you need to allow MPRIS access because I see:

```
[2:1101/173233.329099:ERROR:bus.cc(571)] Failed to get the ownership of org.mpris.MediaPlayer2.chromium.instance2: org.freedesktop.DBus.Error.ServiceUnknown
```

Follow https://docs.flatpak.org/en/latest/sandbox-permissions.html#d-bus-access to find out what is needed, use `--log-session-bus`.